<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Components</class>
 <widget class="QWidget" name="Components">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ui</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_counter">
      <attribute name="title">
       <string>Лічильник</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>Скинути</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="counterButton">
         <property name="text">
          <string>Натиснути</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="counterLabel">
         <property name="font">
          <font>
           <pointsize>64</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_grid">
      <attribute name="title">
       <string>Таблиця</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="2">
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Очистити</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="rowSpinBox">
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rowLabel">
         <property name="text">
          <string>Рядки:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QSpinBox" name="columnSpinBox">
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="5" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <property name="rowCount">
          <number>5</number>
         </property>
         <property name="columnCount">
          <number>5</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="fillButton">
         <property name="text">
          <string>Заповнити</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="columnLabel">
         <property name="text">
          <string>стовпці:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
